src/components/RegionsContainer.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import OneRegion from './OneRegion';
+import './RegionsContainer.css';
+
+const RegionsContainer = () => {
+  const countries = useSelector((state) => state.countriesReducer);
+  const { id } = useParams();
+  let country = [];
+  countries.forEach((element) => {
+    if (element.name === id) {
+      country = element;
+    }
+  });
+
+  const {
+    name,
+    regions,
+    today_confirmed,
+    today_deaths,
+    today_open_cases,
+    today_recovered,
+  } = country;
```

Comment:
- [ ] Kindly ensure that there's a link from the `Detail` page which in your own case is the `RegionsContainer` to the `Home` page which in your own case `CountriesContainer` by clicking the "Back" button.